src/spell_parser.cpp

```diff
@@ -1519,8 +1519,27 @@ ACMD(do_cast) {
         SET_FLAG(GET_EVENT_FLAGS(ch), EVENT_CASTING);
 
         /* Chance to quick chant. */
-        if (random_number(1, 102) < GET_SKILL(ch, SKILL_QUICK_CHANT))
+        if (random_number(1, 110) < (GET_SKILL(ch, SKILL_QUICK_CHANT) + int_app[GET_INT(ch)].bonus + wis_app[GET_WIS(ch)].bonus)) {
+            int maxcircle, *level_assignment, spellcircle;
+
+/*
+            *level_assignment = skills[spellnum].min_level[GET_CLASS(ch)];
```

Comment:
remove the comments and dead code? (unused variables)

Reply:
Done